autoprotocol/builders.py

```diff
@@ -1324,26 +1327,31 @@ def transport(self, volume=None, pump_override_volume=None,
         mode_params : dict, optional
             Mode parameters
             See Also LiquidHandle.builders.mode_params
+        density : Unit or str, optional
+            Density of the liquid to be aspirated/dispensed in mg/ml.
 
 
         Returns
         -------
         dict
             transport parameters for a LiquidHandle instruction
         """
-        if volume is not None:
+        if volume:
```

Comment:
we actually want the stricter checks of `is not None` here since AFAIK `0:microliters` is falsy and therefore wouldn't hit this check

Reply:
oh good to know... thanks!